Translations update from Hosted Weblate

weblate opened the pull request:
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [OpenStop/App](https://hosted.weblate.org/projects/openstop/app/).



Current translation status:

![Weblate translation status](https://hosted.weblate.org/widget/openstop/app/horizontal-auto.svg)

weblate committed:
Translated using Weblate (Belarusian)

Currently translated at 97.6% (329 of 337 strings)

Translated using Weblate (Chinese (Simplified Han script))

Currently translated at 0.0% (0 of 337 strings)

Translated using Weblate (Norwegian Bokmål)

Currently translated at 73.5% (114 of 155 strings)

Translated using Weblate (Italian)

Currently translated at 19.8% (67 of 337 strings)

Translated using Weblate (Norwegian Bokmål)

Currently translated at 21.0% (71 of 337 strings)

Co-authored-by: Anonymous <noreply@weblate.org>
Translate-URL: https://hosted.weblate.org/projects/openstop/app/nb_NO/
Translate-URL: https://hosted.weblate.org/projects/openstop/question-catalog/be/
Translate-URL: https://hosted.weblate.org/projects/openstop/question-catalog/it/
Translate-URL: https://hosted.weblate.org/projects/openstop/question-catalog/nb_NO/
Translate-URL: https://hosted.weblate.org/projects/openstop/question-catalog/zh_Hans/
Translation: OpenStop/App
Translation: OpenStop/Question Catalog

Mannivu committed:
Translated using Weblate (Italian)

Currently translated at 96.7% (150 of 155 strings)

Co-authored-by: Manuel Tassi <mannivuwiki@gmail.com>
Translate-URL: https://hosted.weblate.org/projects/openstop/app/it/
Translation: OpenStop/App

weblate committed:
Translated using Weblate (Swedish)

Currently translated at 65.8% (222 of 337 strings)

Translated using Weblate (Czech)

Currently translated at 89.6% (302 of 337 strings)

Translated using Weblate (Italian)

Currently translated at 96.7% (150 of 155 strings)

Co-authored-by: Anonymous <noreply@weblate.org>
Translate-URL: https://hosted.weblate.org/projects/openstop/app/it/
Translate-URL: https://hosted.weblate.org/projects/openstop/question-catalog/cs/
Translate-URL: https://hosted.weblate.org/projects/openstop/question-catalog/sv/
Translation: OpenStop/App
Translation: OpenStop/Question Catalog

comradekingu committed:
Translated using Weblate (Italian)

Currently translated at 6.2% (1 of 16 strings)

Co-authored-by: Allan Nordhøy <epost@anotheragency.no>
Translate-URL: https://hosted.weblate.org/projects/openstop/advanced-question-catalog/it/
Translation: OpenStop/Advanced Question Catalog

weblate committed:
Translated using Weblate (Croatian)

Currently translated at 21.2% (33 of 155 strings)

Translated using Weblate (Spanish)

Currently translated at 76.7% (119 of 155 strings)

Translated using Weblate (Chinese (Traditional Han script))

Currently translated at 53.7% (181 of 337 strings)

Translated using Weblate (Tamil)

Currently translated at 99.3% (154 of 155 strings)

Translated using Weblate (French)

Currently translated at 75.3% (254 of 337 strings)

Translated using Weblate (Persian)

Currently translated at 34.1% (53 of 155 strings)

Translated using Weblate (Czech)

Currently translated at 75.4% (117 of 155 strings)

Translated using Weblate (Belarusian)

Currently translated at 80.6% (125 of 155 strings)

Translated using Weblate (French)

Currently translated at 75.4% (117 of 155 strings)

Translated using Weblate (Russian)

Currently translated at 99.3% (154 of 155 strings)

Translated using Weblate (Portuguese)

Currently translated at 76.7% (119 of 155 strings)

Translated using Weblate (Persian)

Currently translated at 1.4% (5 of 337 strings)

Translated using Weblate (Chinese (Simplified Han script))

Currently translated at 62.5% (97 of 155 strings)

Translated using Weblate (Croatian)

Currently translated at 4.1% (14 of 337 strings)

Translated using Weblate (Swedish)

Currently translated at 76.7% (119 of 155 strings)

Translated using Weblate (Italian)

Currently translated at 6.2% (1 of 16 strings)

Co-authored-by: Anonymous <noreply@weblate.org>
Translate-URL: https://hosted.weblate.org/projects/openstop/advanced-question-catalog/it/
Translate-URL: https://hosted.weblate.org/projects/openstop/app/be/
Translate-URL: https://hosted.weblate.org/projects/openstop/app/cs/
Translate-URL: https://hosted.weblate.org/projects/openstop/app/es/
Translate-URL: https://hosted.weblate.org/projects/openstop/app/fa/
Translate-URL: https://hosted.weblate.org/projects/openstop/app/fr/
Translate-URL: https://hosted.weblate.org/projects/openstop/app/hr/
Translate-URL: https://hosted.weblate.org/projects/openstop/app/pt/
Translate-URL: https://hosted.weblate.org/projects/openstop/app/ru/
Translate-URL: https://hosted.weblate.org/projects/openstop/app/sv/
Translate-URL: https://hosted.weblate.org/projects/openstop/app/ta/
Translate-URL: https://hosted.weblate.org/projects/openstop/app/zh_Hans/
Translate-URL: https://hosted.weblate.org/projects/openstop/question-catalog/fa/
Translate-URL: https://hosted.weblate.org/projects/openstop/question-catalog/fr/
Translate-URL: https://hosted.weblate.org/projects/openstop/question-catalog/hr/
Translate-URL: https://hosted.weblate.org/projects/openstop/question-catalog/zh_Hant/
Translation: OpenStop/Advanced Question Catalog
Translation: OpenStop/App
Translation: OpenStop/Question Catalog

Commit:
Translated using Weblate (Swedish)

Currently translated at 76.7% (119 of 155 strings)

Co-authored-by: Erik Palm <erik.p.palm@gmail.com>
Translate-URL: https://hosted.weblate.org/projects/openstop/app/sv/
Translation: OpenStop/App

Suburbanno committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (155 of 155 strings)

Co-authored-by: Vinicius <rodriguessv30@gmail.com>
Translate-URL: https://hosted.weblate.org/projects/openstop/app/pt_BR/
Translation: OpenStop/App

Commit:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (337 of 337 strings)

Co-authored-by: F Bausch <florian+weblate@fbausch.de>
Translate-URL: https://hosted.weblate.org/projects/openstop/question-catalog/pt_BR/
Translation: OpenStop/Question Catalog

s8321414 committed:
Translated using Weblate (Chinese (Traditional Han script))

Currently translated at 100.0% (155 of 155 strings)

Co-authored-by: Jeff Huang <s8321414@gmail.com>
Translate-URL: https://hosted.weblate.org/projects/openstop/app/zh_Hant/
Translation: OpenStop/App

7h30n3 committed:
Translated using Weblate (German)

Currently translated at 100.0% (11 of 11 strings)

Co-authored-by: René <18004390+7h30n3@users.noreply.github.com>
Translate-URL: https://hosted.weblate.org/projects/openstop/fastlane/de/
Translation: OpenStop/Fastlane

7h30n3 commented:
Due to a recent Weblate update before our last OpenStop release the indents of updated ARB files were set to 4 spaces. Using the[ file format parameters](https://docs.weblate.org/en/latest/formats.html#file-format-params) introduced in Weblate 5.13 the indents were set to 2 spaces again.

weblate committed:
Translated using Weblate (Belarusian)

Currently translated at 97.6% (329 of 337 strings)

Translated using Weblate (Chinese (Simplified Han script))

Currently translated at 0.0% (0 of 337 strings)

Translated using Weblate (Norwegian Bokmål)

Currently translated at 73.5% (114 of 155 strings)

Translated using Weblate (Italian)

Currently translated at 19.8% (67 of 337 strings)

Translated using Weblate (Norwegian Bokmål)

Currently translated at 21.0% (71 of 337 strings)

Co-authored-by: Anonymous <noreply@weblate.org>
Translate-URL: https://hosted.weblate.org/projects/openstop/app/nb_NO/
Translate-URL: https://hosted.weblate.org/projects/openstop/question-catalog/be/
Translate-URL: https://hosted.weblate.org/projects/openstop/question-catalog/it/
Translate-URL: https://hosted.weblate.org/projects/openstop/question-catalog/nb_NO/
Translate-URL: https://hosted.weblate.org/projects/openstop/question-catalog/zh_Hans/
Translation: OpenStop/App
Translation: OpenStop/Question Catalog

Mannivu committed:
Translated using Weblate (Italian)

Currently translated at 54.5% (6 of 11 strings)

Translated using Weblate (Italian)

Currently translated at 96.7% (150 of 155 strings)

Co-authored-by: Manuel Tassi <mannivuwiki@gmail.com>
Translate-URL: https://hosted.weblate.org/projects/openstop/app/it/
Translate-URL: https://hosted.weblate.org/projects/openstop/fastlane/it/
Translation: OpenStop/App
Translation: OpenStop/Fastlane

comradekingu committed:
Translated using Weblate (Italian)

Currently translated at 6.2% (1 of 16 strings)

Co-authored-by: Allan Nordhøy <epost@anotheragency.no>
Translate-URL: https://hosted.weblate.org/projects/openstop/advanced-question-catalog/it/
Translation: OpenStop/Advanced Question Catalog

weblate committed:
Translated using Weblate (Swedish)

Currently translated at 65.8% (222 of 337 strings)

Translated using Weblate (Czech)

Currently translated at 89.6% (302 of 337 strings)

Translated using Weblate (Italian)

Currently translated at 96.7% (150 of 155 strings)

Co-authored-by: Anonymous <noreply@weblate.org>
Translate-URL: https://hosted.weblate.org/projects/openstop/app/it/
Translate-URL: https://hosted.weblate.org/projects/openstop/question-catalog/cs/
Translate-URL: https://hosted.weblate.org/projects/openstop/question-catalog/sv/
Translation: OpenStop/App
Translation: OpenStop/Question Catalog

weblate committed:
Translated using Weblate (Croatian)

Currently translated at 21.2% (33 of 155 strings)

Translated using Weblate (Spanish)

Currently translated at 76.7% (119 of 155 strings)

Translated using Weblate (Chinese (Traditional Han script))

Currently translated at 53.7% (181 of 337 strings)

Translated using Weblate (Tamil)

Currently translated at 99.3% (154 of 155 strings)

Translated using Weblate (French)

Currently translated at 75.3% (254 of 337 strings)

Translated using Weblate (Persian)

Currently translated at 34.1% (53 of 155 strings)

Translated using Weblate (Czech)

Currently translated at 75.4% (117 of 155 strings)

Translated using Weblate (Belarusian)

Currently translated at 80.6% (125 of 155 strings)

Translated using Weblate (French)

Currently translated at 75.4% (117 of 155 strings)

Translated using Weblate (Russian)

Currently translated at 99.3% (154 of 155 strings)

Translated using Weblate (Portuguese)

Currently translated at 76.7% (119 of 155 strings)

Translated using Weblate (Persian)

Currently translated at 1.4% (5 of 337 strings)

Translated using Weblate (Chinese (Simplified Han script))

Currently translated at 62.5% (97 of 155 strings)

Translated using Weblate (Croatian)

Currently translated at 4.1% (14 of 337 strings)

Translated using Weblate (Swedish)

Currently translated at 76.7% (119 of 155 strings)

Translated using Weblate (Italian)

Currently translated at 6.2% (1 of 16 strings)

Co-authored-by: Anonymous <noreply@weblate.org>
Translate-URL: https://hosted.weblate.org/projects/openstop/advanced-question-catalog/it/
Translate-URL: https://hosted.weblate.org/projects/openstop/app/be/
Translate-URL: https://hosted.weblate.org/projects/openstop/app/cs/
Translate-URL: https://hosted.weblate.org/projects/openstop/app/es/
Translate-URL: https://hosted.weblate.org/projects/openstop/app/fa/
Translate-URL: https://hosted.weblate.org/projects/openstop/app/fr/
Translate-URL: https://hosted.weblate.org/projects/openstop/app/hr/
Translate-URL: https://hosted.weblate.org/projects/openstop/app/pt/
Translate-URL: https://hosted.weblate.org/projects/openstop/app/ru/
Translate-URL: https://hosted.weblate.org/projects/openstop/app/sv/
Translate-URL: https://hosted.weblate.org/projects/openstop/app/ta/
Translate-URL: https://hosted.weblate.org/projects/openstop/app/zh_Hans/
Translate-URL: https://hosted.weblate.org/projects/openstop/question-catalog/fa/
Translate-URL: https://hosted.weblate.org/projects/openstop/question-catalog/fr/
Translate-URL: https://hosted.weblate.org/projects/openstop/question-catalog/hr/
Translate-URL: https://hosted.weblate.org/projects/openstop/question-catalog/zh_Hant/
Translation: OpenStop/Advanced Question Catalog
Translation: OpenStop/App
Translation: OpenStop/Question Catalog

Commit:
Translated using Weblate (Swedish)

Currently translated at 76.7% (119 of 155 strings)

Co-authored-by: Erik Palm <erik.p.palm@gmail.com>
Translate-URL: https://hosted.weblate.org/projects/openstop/app/sv/
Translation: OpenStop/App

Commit:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (337 of 337 strings)

Co-authored-by: F Bausch <florian+weblate@fbausch.de>
Translate-URL: https://hosted.weblate.org/projects/openstop/question-catalog/pt_BR/
Translation: OpenStop/Question Catalog

Suburbanno committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (155 of 155 strings)

Co-authored-by: Vinicius <rodriguessv30@gmail.com>
Translate-URL: https://hosted.weblate.org/projects/openstop/app/pt_BR/
Translation: OpenStop/App

s8321414 committed:
Translated using Weblate (Chinese (Traditional Han script))

Currently translated at 100.0% (11 of 11 strings)

Translated using Weblate (Chinese (Traditional Han script))

Currently translated at 100.0% (155 of 155 strings)

Co-authored-by: Jeff Huang <s8321414@gmail.com>
Translate-URL: https://hosted.weblate.org/projects/openstop/app/zh_Hant/
Translate-URL: https://hosted.weblate.org/projects/openstop/fastlane/zh_Hant/
Translation: OpenStop/App
Translation: OpenStop/Fastlane

7h30n3 committed:
Translated using Weblate (Czech)

Currently translated at 75.4% (117 of 155 strings)

Translated using Weblate (Russian)

Currently translated at 100.0% (155 of 155 strings)

Translated using Weblate (Norwegian Bokmål)

Currently translated at 73.5% (114 of 155 strings)

Translated using Weblate (German)

Currently translated at 100.0% (11 of 11 strings)

Co-authored-by: René <18004390+7h30n3@users.noreply.github.com>
Translate-URL: https://hosted.weblate.org/projects/openstop/app/cs/
Translate-URL: https://hosted.weblate.org/projects/openstop/app/nb_NO/
Translate-URL: https://hosted.weblate.org/projects/openstop/app/ru/
Translate-URL: https://hosted.weblate.org/projects/openstop/fastlane/de/
Translation: OpenStop/App
Translation: OpenStop/Fastlane

matheusgomesms committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (11 of 11 strings)

Co-authored-by: Matheus Gomes Correia <matheus.gomes03@hotmail.com>
Translate-URL: https://hosted.weblate.org/projects/openstop/fastlane/pt_BR/
Translation: OpenStop/Fastlane

assanges committed:
Translated using Weblate (Chinese (Traditional Han script))

Currently translated at 100.0% (337 of 337 strings)

Co-authored-by: Sean Young <assanges@gmail.com>
Translate-URL: https://hosted.weblate.org/projects/openstop/question-catalog/zh_Hant/
Translation: OpenStop/Question Catalog

s8321414 committed:
Translated using Weblate (Chinese (Traditional Han script))

Currently translated at 100.0% (337 of 337 strings)

Co-authored-by: Jeff Huang <s8321414@gmail.com>
Translate-URL: https://hosted.weblate.org/projects/openstop/question-catalog/zh_Hant/
Translation: OpenStop/Question Catalog

debyos committed:
Translated using Weblate (Hungarian)

Currently translated at 3.2% (5 of 155 strings)

Translated using Weblate (Hungarian)

Currently translated at 27.2% (3 of 11 strings)

Translated using Weblate (Hungarian)

Currently translated at 1.4% (5 of 337 strings)

Translated using Weblate (Hungarian)

Currently translated at 31.2% (5 of 16 strings)

Added translation using Weblate (Hungarian)

Added translation using Weblate (Hungarian)

Added translation using Weblate (Hungarian)

Co-authored-by: Babos Gábor <gabor.babos@gmail.com>
Translate-URL: https://hosted.weblate.org/projects/openstop/advanced-question-catalog/hu/
Translate-URL: https://hosted.weblate.org/projects/openstop/app/hu/
Translate-URL: https://hosted.weblate.org/projects/openstop/fastlane/hu/
Translate-URL: https://hosted.weblate.org/projects/openstop/question-catalog/hu/
Translation: OpenStop/Advanced Question Catalog
Translation: OpenStop/App
Translation: OpenStop/Fastlane
Translation: OpenStop/Question Catalog

TamilNeram committed:
Translated using Weblate (Tamil)

Currently translated at 100.0% (155 of 155 strings)

Translated using Weblate (Tamil)

Currently translated at 100.0% (11 of 11 strings)

Co-authored-by: தமிழ்நேரம் <tamilneram247@gmail.com>
Translate-URL: https://hosted.weblate.org/projects/openstop/app/ta/
Translate-URL: https://hosted.weblate.org/projects/openstop/fastlane/ta/
Translation: OpenStop/App
Translation: OpenStop/Fastlane

ivanbranco committed:
Translated using Weblate (Italian)

Currently translated at 25.0% (4 of 16 strings)

Translated using Weblate (Italian)

Currently translated at 27.0% (91 of 337 strings)

Co-authored-by: Ivan R. <ivanrugg@gmail.com>
Translate-URL: https://hosted.weblate.org/projects/openstop/advanced-question-catalog/it/
Translate-URL: https://hosted.weblate.org/projects/openstop/question-catalog/it/
Translation: OpenStop/Advanced Question Catalog
Translation: OpenStop/Question Catalog

SantosSi committed:
Translated using Weblate (Portuguese)

Currently translated at 100.0% (155 of 155 strings)

Translated using Weblate (Portuguese)

Currently translated at 100.0% (337 of 337 strings)

Co-authored-by: ssantos <ssantos@web.de>
Translate-URL: https://hosted.weblate.org/projects/openstop/app/pt/
Translate-URL: https://hosted.weblate.org/projects/openstop/question-catalog/pt/
Translation: OpenStop/App
Translation: OpenStop/Question Catalog

Goudarz committed:
Translated using Weblate (Persian)

Currently translated at 35.4% (55 of 155 strings)

Co-authored-by: Goudarz Jafari <me@goudarzjafari.com>
Translate-URL: https://hosted.weblate.org/projects/openstop/app/fa/
Translation: OpenStop/App

bounkirdni-2025 committed:
Added translation using Weblate (Lao)

Added translation using Weblate (Lao)

Added translation using Weblate (Lao)

Co-authored-by: BoneNI <bounkirdni@gmail.com>

Commit:
Translated using Weblate (Italian)

Currently translated at 57.8% (195 of 337 strings)

Translated using Weblate (Italian)

Currently translated at 100.0% (155 of 155 strings)

Translated using Weblate (Italian)

Currently translated at 81.2% (13 of 16 strings)

Co-authored-by: Antonella <g.antonella5@gmail.com>
Translate-URL: https://hosted.weblate.org/projects/openstop/advanced-question-catalog/it/
Translate-URL: https://hosted.weblate.org/projects/openstop/app/it/
Translate-URL: https://hosted.weblate.org/projects/openstop/question-catalog/it/
Translation: OpenStop/Advanced Question Catalog
Translation: OpenStop/App
Translation: OpenStop/Question Catalog

Commit:
Translated using Weblate (Polish)

Currently translated at 13.9% (47 of 337 strings)

Translated using Weblate (Polish)

Currently translated at 18.1% (2 of 11 strings)

Translated using Weblate (Polish)

Currently translated at 85.8% (133 of 155 strings)

Translated using Weblate (Polish)

Currently translated at 62.5% (10 of 16 strings)

Added translation using Weblate (Polish)

Added translation using Weblate (Polish)

Added translation using Weblate (Polish)

Co-authored-by: Hosted Weblate user 159754 <kubus.lukasiak@users.noreply.hosted.weblate.org>
Translate-URL: https://hosted.weblate.org/projects/openstop/advanced-question-catalog/pl/
Translate-URL: https://hosted.weblate.org/projects/openstop/app/pl/
Translate-URL: https://hosted.weblate.org/projects/openstop/fastlane/pl/
Translate-URL: https://hosted.weblate.org/projects/openstop/question-catalog/pl/
Translation: OpenStop/Advanced Question Catalog
Translation: OpenStop/App
Translation: OpenStop/Fastlane
Translation: OpenStop/Question Catalog

7h30n3 committed:
Translated using Weblate (Polish)

Currently translated at 85.8% (133 of 155 strings)

Co-authored-by: René <18004390+7h30n3@users.noreply.github.com>
Translate-URL: https://hosted.weblate.org/projects/openstop/app/pl/
Translation: OpenStop/App
